storage.go

```diff
@@ -242,8 +228,11 @@ func (s *Storage) Lock(ctx context.Context, key string) error {
 	lockKey := fmt.Sprintf("LOCK-%s", key)
 
 	// Check for existing lock
+	var it Item
```

Comment:
`nit` I think just calling it `item` is better

Reply:
sure